### PR TITLE
Remove non-CF build files, solutions and projects

### PR DESCRIPTION
**Pull request by rprouse**
Fixes nunit/nunit#1599

### CODE REVIEW

**Comment by rprouse** [comment]
This is ready to review. In this repo, I have locked down master like in the other repositories and made it dependent on AppVeyor. AppVeyor is setup and building and I checked the artifacts.

This PR removes all non-CF project files and solutions and fixed the build.cake file to build and package only CF. This PR does not remove any of the #if statements in the code. I leave that up to the future maintainer at their discretion.

**Comment by rprouse** [comment]
Not sure who got notified for this PR since I added the @nunit/core-team to the repo after creating the PR. @CharliePoole or @ChrisMaddock can you take a look when you get a chance?

File: build.cake
Comment (CharliePoole) [inline build.cake]: Change to build?

File: build.cake
Comment (CharliePoole) [inline build.cake]: Remove?

**Review by CharliePoole — APPROVED** [review]
Looks good to me. I made a few line comments and I think there would be others to make about build.cake if I looked at the unchanged part. However, these changes can be made later as a separate thing or perhaps as part of standardizing our small (single assembly) builds.

**Comment by CharliePoole** [comment]
I want to think about how much cleanup we should do on the two extracted framework builds, if any. Leaving it for whomever takes over is certainly one aproach but there could be some value in getting it a bit cleaner and ready for taking over ahead of time.

File: build.cake
Comment (rprouse) [reply build.cake]: I did that in the SL, so I will do the same here.

**Comment by rprouse** [comment]
@CharliePoole I've made the changes you've requested and a few others that I found while searching through the file. I'd already deleted all unused variables and methods and cleaned up in general, I just missed a couple of things.

**Comment by ChrisMaddock** [comment]
All looks good, Rob! There's just a stray comma in the repo description, that I don't have access to fix myself, would yourself or @CharliePoole mind?

> The ,NET Compact Framework 3.5 version of the NUnit Framework

**Comment by CharliePoole** [comment]
Fixed

**Pull request merged**
